### DISCUSSION TITLE
Application icons

### DISCUSSION THREAD

**Discussion by xlebpushek** [davatorium/rofi#1521] [Q&A] [answered]
How does rofi look for an icon for an application? What path is rofi looking for (file) and what is the name of the file itself?

**Comment by DaveDavenport** [comment, marked as answer]
That depends on the modi:
* drun -> icon is specified in desktop file (either name or full path)
* window ->  queries the raw icon from XServer directly, or queries it based on name.
* run -> queries it based on name.

A name is resolved using the icon theme. This is an XDG standard.

**Reply by xlebpushek** [reply]
Thx
